src/main/java/com/laborhelper/enterprise/LaborHelperController.java

```diff
@@ -8,11 +8,11 @@ public class LaborHelperController {
 
     /**
      * Handle the root (/) endpoint and return a start page
-     * @return
+     * @return This method returns an HTML page
```

Comment:
I'm not sure why but the Spring Initializr, does not include a value for the return parameter, so I added one for you. I would recommend adding some text for the return param